webanno-api-dao/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/dao/AnnotationSchemaServiceImpl.java

```diff
@@ -456,7 +475,7 @@ public AnnotationLayer getLayer(Project aProject, FeatureStructure aFS)
             if (layerName.endsWith("Link")) {
                 layerName = layerName.substring(0, layerName.length() - 4);
             }
-            layer = getLayer(layerName, aProject);
+            layer = findLayer(aProject, layerName);
```

Comment:
The indentation above looks really sketchy

Reply:
Looking at the code, I don't see what you refer to.

Reply:
it looked strange due to github hiding lines, my fault :(